go/chat_pub.go

```diff
@@ -0,0 +1,3 @@
+channel := client.Channels.Get("room:tesla-fans")
+channel.Presence.Enter(context.Background(), UserName + " entered the channel")
```

Comment:
Please use `ctx` rather than `context.Background()` to save some horizontal space